src/targets/services/categorize.js

```diff
@@ -4,48 +4,43 @@ global.btoa = require('btoa')
 import fs from 'fs'
 import CozyClient, { Q } from 'cozy-client'
 import { BANK_DOCTYPE } from '../../doctypes'
-
 import { Model } from './helpers'
+import dissecConfig from '../../../dissec.config.json'
 
 export const categorize = async () => {
-  const { pretrained } = JSON.parse(process.env['COZY_PAYLOAD'] || "{ \"pretrained\": false }")
-
-  // eslint-disable-next-line no-console
-  console.log('Passed arguments:', pretrained)
+  const { pretrained } = JSON.parse(process.env['COZY_PAYLOAD'] || {})
 
   const client = CozyClient.fromEnv(process.env, {})
 
   // 1. Fetch data
   const { data: operations } = await client.query(Q(BANK_DOCTYPE))
-  console.log('Found', operations.length, 'operations')
 
   // 2. Fetch model or initialize it
   let model
   if (pretrained) {
     // Use the stack's remote assets
     try {
       const backup = JSON.parse(
-        fs.readFileSync('/mnt/c/Users/Projets/Cozy/categorization-model/model.json')
+        fs.readFileSync(
+          dissecConfig.localModelPath
+        )
       )
       model = Model.fromBackup(backup)
     } catch (err) {
-      console.log('Failed opening backup', err)
       model = Model.fromDocs(operations)
```

Comment:
I don't think you should silently switch  when the pretrained model is not found. I would prefer an actual failing like `throw 'Local model not found at ', dissecConfig.localModelPath`. 
Also, it would be nice to add a small doc about this file in the readme :)

Reply:
Since services don't send users feedback, crashing and logging it will still be silent for the user. Why is a hard failure preferable?

Reply:
Moreover, the first model is created by the final aggregation. Before that, the execution would systematically fail, preventing reaching the final aggregation. 

A solution would be to create a base model manually and add it to the git

Reply:
> Since services don't send users feedback, crashing and logging it will still be silent for the user. Why is a hard failure preferable?

This is not about user here, but about the service execution itself. When a service throws an error, this is logged by the stack  and we can monitor it later. If you don't throw an error, you might have a bug that will be never caught. Here, if I run the service on my computer without editing the config file, I will never notice that I don't have the correct path

Reply:
> A solution would be to create a base model manually and add it to the git

That seems legit